cypress/integration/journalistCanCreateArticles.feature.js

```diff
@@ -1,96 +1,97 @@
+describe('Journalist can create articles', () => {
   beforeEach(() => {
     cy.server();
     cy.route({
-      method: "POST",
-      url: "http://localhost:3000/api/auth/sign_in",
-      response: "fixture:journalist_can_login.json",
+      method: 'POST',
+      url: 'http://localhost:3000/api/auth/sign_in',
+      response: 'fixture:journalist_can_login.json',
       headers: {
-        uid: "journalist@mail.com",
+        uid: 'journalist@mail.com',
       },
     });
     cy.route({
-      method: "GET",
-      url: "http://localhost:3000/api/auth/validate_token**",
-      response: "fixture:journalist_can_login.json",
+      method: 'GET',
+      url: 'http://localhost:3000/api/auth/validate_token**',
+      response: 'fixture:journalist_can_login.json',
     });
     cy.route({
-      method: "POST",
-      url: "http://localhost:3000/api/articles",
-      response: { message: "Your article was successfully created!" },
+      method: 'POST',
+      url: 'http://localhost:3000/api/articles',
+      response: { message: 'Your article was successfully created!' },
     });
-    cy.visit("/");
+    cy.visit('/');
   });
 
-  it("successfully", () => {
+  it('successfully', () => {
     cy.get("[data-cy='login-form']").within(() => {
-      cy.get("[data-cy='email']").type("journalist@mail.com");
-      cy.get("[data-cy='password']").type("password");
-      cy.get("[data-cy='submit-btn']").contains("Submit").click();
+      cy.get("[data-cy='email']").type('journalist@mail.com');
+      cy.get("[data-cy='password']").type('password');
+      cy.get("[data-cy='submit-btn']").contains('Submit').click();
     });
     cy.get('[data-cy="create-article-form"]').within(() => {
-      cy.get('[data-cy="input-title"]').type("Covid vaccine is found");
+      cy.get('[data-cy="input-title"]').type('Covid vaccine is found');
       cy.get('[data-cy="input-sub-title"]').type(
-        "Each person should receive two doses of vaccine"
+        'Each person should receive two doses of vaccine'
       );
       cy.get('[data-cy="input-content"]').type(
-        "Russia has distributed vaccines months ago"
+        'Russia has distributed vaccines months ago'
       );
+      cy.get('[name="file_input"]').attachFile('image.png');
       cy.get('[data-cy="create-article-button"]').click();
     });
     cy.get('[data-cy="response-message"]').should(
-      "contain",
-      "Your article was successfully created!"
+      'contain',
+      'Your article was successfully created!'
     );
   });
 
-  describe("unsuccessfully when input field is not filled in", () => {
+  describe('unsuccessfully when input field is not filled in', () => {
     beforeEach(() => {
       cy.route({
-        method: "POST",
-        url: "http://localhost:3000/api/articles",
-        response: { message: "Something went wrong!" },
+        method: 'POST',
+        url: 'http://localhost:3000/api/articles',
+        response: { message: 'Please fill in all input fields!' },
```

Comment:
Is this really the message that you are sending from the api?

Reply:
It wasn’t previously, but the plan was to change it during the corresponding api user story